docs/en/index.rst

```diff
@@ -1,10 +1,22 @@
 Event Manager Documentation
 ===========================
 
-Welcome to the Doctrine Event Manager Library documentation.
+The Doctrine Event Manager documentation is a reference guide to everything you need
+to know about the project.
 
-.. toctree::
-   :depth: 2
-   :glob:
+Getting Help
+------------
 
-   *
+If this documentation is not helping to answer questions you have about the
+Doctrine DBAL, don't panic. You can get help from different sources:
+
+-  Gitter chat room `#doctrine/event-manager <https://gitter.im/doctrine/event-manager>`_
```

Comment:
Do we actually have gitter rooms per repo?

Reply:
Yes, I have been creating them. I will create it for this project.